examples/pytorch/gin/README.md

```diff
@@ -28,18 +28,58 @@ An experiment on the GIN in customized settings can be run with
 python main.py [--device 0 | --disable-cuda] --dataset COLLAB \
                --graph_pooling_type max --neighbor_pooling_type sum
 ```
+add `--degree_as_nlabel` to use one-hot encodings of node degrees as node feature vectors
 
 Results
 -------
 
 Run with following with the double SUM pooling way:
-(tested dataset: "MUTAG"(default), "COLLAB", "IMDBBINARY", "IMDBMULTI")
+(tested dataset: "MUTAG"(default), "COLLAB", "IMDBBINARY", "IMDBMULTI"), results may **fluctuate**, due to random factors and the relatively small data set.
```

Comment:
You can simply remove "Run with following with the double SUM pooling way: (tested dataset: "MUTAG"(default), "COLLAB", "IMDBBINARY", "IMDBMULTI"),"